ngraph/core/shape_inference/include/shape_infer_utils.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+#include <ngraph/partial_shape.hpp>
+
+namespace ShapeInfer {
```

Comment:
Seems it could be [here](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/shape_inference/include/utils.hpp)?

Reply:
fix in 4a44503e12d519ae51cc7989ffc24bce78c037a7